Dockerfiles/cpp/.devcontainer/Dockerfile

```diff
@@ -105,8 +105,10 @@ USER 4000
 RUN ./install-k3d-tooling.sh
 
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-cpp-sdk/main/requirements.txt  /container/requirements.txt
+COPY --chown=4000 ./requirements-common.txt /container/requirements-common.txt
```

Comment:
Couldn't think for better name than "requirements-common". Did not want to use just "requirements" for avoiding name clashes.